### PR TITLE
MM-41280 - refactor the payment page layout so the content fits well

### PR DESCRIPTION
**Pull request by pablovelezvidal**
#### Summary
This PR adds the code to refactor the payment page layout so the content fits well  no matter the language.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-41280

#### Screenshots
Captura de pantalla 2022-02-16 a las 13 15 55
Captura de pantalla 2022-02-16 a las 13 16 22
Captura de pantalla 2022-02-16 a las 13 16 45
Captura de pantalla 2022-02-16 a las 13 17 14
Captura de pantalla 2022-02-16 a las 13 28 58

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by JulienTant — COMMENTED** [review]
Good job, LGTM!

**Review by JulienTant — APPROVED** [review]
Good job, LGTM!

File: components/purchase_modal/icon_message.scss [hunk 1]
```diff
@@ -91,5 +79,14 @@
             line-height: 14px;
             text-align: center;
         }
+
+        svg {
+            align-self: center;
+            margin-left: 60px;
+
+            @media screen and (max-width: 600px) {
+                width: 150%;
```
Comment (julmondragon) [inline components/purchase_modal/icon_message.scss:88]: I think setting `max-width: 150%` to the element will work as well without needing the media query.

**Review by julmondragon — APPROVED** [review]
Looks good!

**Comment by stevemudie** [comment]
@pablovelezvidal I'm not sure why but it doesn't appear any different to me even when testing in an incognito window.

The button is still far to the left just like in the ticket screenshots in languages other than English. 🤔

**Comment by pablovelezvidal** [comment]
> @pablovelezvidal I'm not sure why but it doesn't appear any different to me even when testing in an incognito window.
> 
> The button is still far to the left just like in the ticket screenshots in languages other than English. 🤔

weird, let me check! I will let you know when I figure it out. Thanks @stevemudie

**Review by stevemudie — APPROVED** [review]
Much better. Thanks for the quick turnaround @pablovelezvidal!

The buttons are centered as expected. This is all set.

**Pull request merged**